simulationTool/components/HelpSection/HelpPanel.vue

```diff
@@ -0,0 +1,199 @@
+                },
+                {
+                    question: this.$t('additional:modules.tools.simulationTool.QstoredData'),
+                    answer: this.$t('additional:modules.tools.simulationTool.AstoredData')
+                },
+                {
+                    question: this.$t('additional:modules.tools.simulationTool.QsharedData'),
+                    answer: this.$t('additional:modules.tools.simulationTool.AsharedData')
+                }
+            ]
+        };
+    },
+    computed: {
+    },
+    mounted() {
+    },
+    watch: {
+    },
+    methods: {
+        ...mapMutations("Modules/SimulationTool", [
+            "setMode"
+        ]),
+    }
+};
+
+</script>
+
+<template>
+    <div class="help-panel">
+        <SectionHeader :title="$t('additional:modules.tools.simulationTool.help')" icon="bi bi-question-circle-fill" />
+
+        <!-- ToDo: implement Searchbar -->
+        <div class="segment-wrapper">
+            <div class="segment-wrapper">
+                <div class="segment-header bold">
+                    {{ $t('additional:modules.tools.simulationTool.tutorial') }}
+                </div>
+                <button class="segment-wrapper btn-light" @click="this.setMode('tutorial-panel')">
+                    <p>{{ $t("additional:modules.tools.simulationTool.tutorialInfo") }}</p>
+                    <i class="bi bi-arrow-right-circle-fill"></i>
+                </button>
+            </div>
+            <div class="segment-wrapper">
+                <div class="segment-header bold">
+                    {{ $t('additional:modules.tools.simulationTool.glossar') }}
+                </div>
+                {{ $t('additional:modules.tools.simulationTool.definitions') }}
+                <FaqItem v-for="(def, index) in definitions" :key="index" :question="def.question"
+                    :answer="def.answer" />
+            </div>
+            <div class="segment-wrapper">
+                <div class="segment-header bold">
+                    {{ $t('additional:modules.tools.simulationTool.faq') }}
+                </div>
+                {{ $t('additional:modules.tools.simulationTool.application') }}
+                <FaqItem v-for="(faq, index) in faqsApplication" :key="index" :question="faq.question"
+                    :answer="faq.answer" />
+                {{ $t('additional:modules.tools.simulationTool.privacy') }}
+                <FaqItem v-for="(faq, index) in faqsPrivacy" :key="index" :question="faq.question"
+                    :answer="faq.answer" />
+            </div>
+            <div class="segment-wrapper">
+                <div class="segment-header bold">
+                    {{ $t('additional:modules.tools.simulationTool.contact') }}
+                </div>
+                <span>
+                    {{ $t('additional:modules.tools.simulationTool.contactInfo') }}
+                    <a href="mailto:cut@hcu-hamburg.de">cut@hcu-hamburg.de</a>
+                    !
```

Comment:
```suggestion
```

Reply:
The ''!" was intentional to close the sentence